WooCommerce/Classes/ViewRelated/Orders/Order Details/Customer Section/Billing Information/BillingInformationViewController.swift

```diff
@@ -93,6 +93,19 @@ private extension BillingInformationViewController {
             tableView.register(kind.loadNib(), forHeaderFooterViewReuseIdentifier: kind.reuseIdentifier)
         }
     }
+
+    /// Presents EditAddressForm modal view
+    ///
+    func editBillingAddress() {
+        let viewModel = EditAddressFormViewModel(order: order, type: .billing) { [weak self] updatedOrder in
+            self?.order = updatedOrder
```

Comment:
I wonder if we should refactor this screen to observe the order thru core data? 🤔 
Maybe add it as a "debt" card?

Reply:
We can also add method like `update(order:)` and pass code data changes from order details VC. Or own core data logic would be better?

Reply:
While I like about the core data observations is that there is no way that one screen could get outdated because someone forgot to call a method or similar.